### PR TITLE
Added Debug option, Playlist listing command, and updated dependencies 

### PR DESCRIPTION
**Pull request by PostsDesert**
Small PR with a few changes. 

Added Debug option, Playlist listing command, and updated dependencies.

### CODE REVIEW

**Comment by stevenleeg** [comment]
Is this missing some commits? Looks like the only diff is the readme change.

**Comment by PostsDesert** [comment]
Yeah, sorry about that. I must have stashed them or something.

File: geemusic/__init__.py [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 
 from .utils.music import GMusicWrapper
 from .utils.music_queue import MusicQueue
 
 app = Flask(__name__)
 ask = Ask(app, '/alexa')
-logging.getLogger("flask_ask").setLevel(logging.INFO)
+
+if str(environ['DEBUG_MODE']) is True:
+    logLevel = logging.DEBUG
```
Comment (stevenleeg) [inline geemusic/__init__.py]: nitpicking, but could you use `snake_case` for these variables to be consistent with the rest of the codebase?

File: geemusic/intents/playback.py [hunk 2]
```diff
@@ -153,6 +154,30 @@ def currently_playing():
                        large_image_url=thumbnail)
 
 
+@ask.intent('GeeMusicListAllPlaylists')
+def listAllPlaylists():
```
Comment (stevenleeg) [inline geemusic/intents/playback.py]: `list_all_playlists`

File: geemusic/intents/playback.py [hunk 3]
```diff
@@ -153,6 +154,30 @@ def currently_playing():
...
+    all_playlists = api.get_all_user_playlist_contents()
+    playlist_names = []
+    for i, match in enumerate(all_playlists):
+
+        playlist_names.append(match['name'])
+        totalPlaylists = i + 1
+
+    # Adds "and" before the last playlist to sound more natural when speaking
+    if len(playlist_names) >= 3:
+        andPlacement = len(playlist_names) - 1
```
Comment (stevenleeg) [inline geemusic/intents/playback.py]: `and_placement`

**Pull request merged**